src/NewProject/add_constraints_form.cpp

```diff
@@ -22,6 +26,17 @@ addConstraintsForm::addConstraintsForm(QWidget *parent)
 
   ui->m_ckkBoxCopy->setText(tr("Copy sources into project."));
   ui->m_ckkBoxCopy->setCheckState(Qt::CheckState::Checked);
+
+  Compiler *compiler = GlobalSession->GetCompiler();
+  if (compiler->PinAssignOpts() == Compiler::PinAssignOpt::Random) {
+    ui->select_random->setChecked(true);
+  } else
```

Comment:
please put {} here or remove for **if** - for consistency

Reply:
Okay

Reply:
Done.